hardhat/contracts/Players.sol

```diff
@@ -78,13 +78,17 @@ contract Players {
         {_shipId} the ship you are moving
      */
     function moveShip(uint x, uint y, uint _planetId, uint _shipId, uint _worldId) public {
-        // TODO: calc distance used
 
         // current location of the ship
         (uint xCoordShip, uint yCoordShip) = nftContract.getLocation(_shipId);
 
+        // calculate distance moved
+        uint travelX = get_abs_diff(xCoordShip, x);
+        uint travelY = get_abs_diff(yCoordShip, y);
+        uint travelDistance = uint(sqrt((travelX * travelX) + (travelY * travelY)));
+        
```

Comment:
do we need to throw a error if you move more steps then u have available? In the frontend we i think we travelDistance * 10 = the steps to substract. Maybe we can have a shared constant for this